docs/roadmap.md

```diff
@@ -51,9 +52,12 @@ clustered database or key-value store. We will target such workloads for our
 1.0 release.
 
 ## v1 APIs 
+
 For existing and future workloads, we want to provide a consistent, stable set of APIs, over which developers can build and extend Kubernetes. This includes input validation, a consistent API structure, clean semantics, and improved diagnosability of the system. 
 ||||||| merged common ancestors
```

Comment:
This is from a merge conflict WEEKS ago.  Sigh.  Anyone have bandwidth to figure out our intentions here :)